src/features/form/containers/RepeatingGroupsEditContainer.tsx

```diff
@@ -27,6 +38,38 @@ export interface IRepeatingGroupsEditContainer {
 }
 
 export function RepeatingGroupsEditContainer({
+  id,
+  editIndex,
+  ...props
+}: IRepeatingGroupsEditContainer): JSX.Element | null {
+  const language = useAppSelector((state) => state.language.language);
+  const node = useResolvedNode(id);
+  const group = node?.item;
+
+  if (!group || !node || group.type !== 'Group' || !('rows' in group)) {
+    return null;
+  }
+
+  const row = group.rows[editIndex];
+
+  if (!row || !language) {
+    return null;
+  }
+
+  return (
+    <RepeatingGroupsEditContainerInternal
+      id={id}
+      editIndex={editIndex}
+      group={group}
+      row={row}
+      node={node}
+      language={language}
+      {...props}
+    />
+  );
+}
+
+export function RepeatingGroupsEditContainerInternal({
```

Comment:
We should remove export on this method to ensure the functional component do not be used outside of this file.
```suggestion
function RepeatingGroupsEditContainerInternal({
```